exportReport/Utility/SharedFunctions.java

```diff
@@ -0,0 +1,183 @@
+package org.zaproxy.zap.extension.exportReport.Utility;
+
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+
+import java.awt.*;
+import java.awt.font.*;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ * CLASS		:	SharedFunctions.java 
+ * DESC			:	Methods that are used frequently to create swing components, etc.
+ * CREATED ON	:	MARCH 10TH, 2016
+ * CURRENT VER	:	V1.0
+ * SOURCE		:	https://github.com/JordanGS/workspace/tree/master/zap-extensions/src/org/zaproxy/zap/extension/exportReport
+ */
+
+/* 
+ * MODIFED BY	:	<NAME> - <GIT USER>
+ * MOD DATE		:	
+ * MOD VERSION	:	<VERSION OF PLUGIN>
+ * MOD DESC		:	
+ */
+
+public class SharedFunctions
+{
+
+	public SharedFunctions()
+	{
+		// TODO Auto-generated constructor stub
+	}
+
+	public static String getCurrentTimeStamp()
+	{
+		SimpleDateFormat date = new SimpleDateFormat("EEE, d MMM yyyy HH:mm:ss z(Z)");
+		Date now = new Date();
+		String strDate = date.format(now);
+		// System.out.println(strDate);
+		return strDate;
+	}
+
+	public static Font getTitleFont()
+	{
+		Map<TextAttribute, Integer> fontAttributes = new HashMap<TextAttribute, Integer>();
+		fontAttributes.put(TextAttribute.UNDERLINE, TextAttribute.UNDERLINE_ON);
+		return new Font("Arial", Font.BOLD, 30).deriveFont(fontAttributes);
```

Comment:
No it does not, i will write a method to check and default to the appropriate Font.